.github/workflows/go-test.yml

```diff
@@ -33,13 +33,6 @@ jobs:
         uses: protocol/multiple-go-modules@v1.2
         with:
           run: go test -v -coverprofile module-coverage.txt ./...
-      - name: Run tests (32 bit)
```

Comment:
I removed this since I don't think we actually need it. It adds a significant latency to run all the tests again in 32bit.

Reply:
The problems with is that the file is auto-generated, so next time there is a change to the CI stuff this gets undone.  Hence the comment at the top:
```go
# File managed by web3-bot. DO NOT EDIT.
```

I might be better to have explicit skips in the tests that fail, like what is done [here](https://github.com/filecoin-project/go-indexer-core/blob/main/store/pogreb/pogreb_test.go#L32) and [here](https://github.com/filecoin-project/go-indexer-core/blob/main/store/pogreb/pogreb_test.go#L87-L91)